synapse/python_dependencies.py

```diff
@@ -48,6 +48,9 @@
         "Jinja2>=2.8": ["Jinja2>=2.8"],
         "bleach>=1.4.2": ["bleach>=1.4.2"],
     },
+    "ldap": {
+        "ldap3": ["ldap3>=1.0"],
```

Comment:
I think you probably want `"ldap3>=1.0": ["ldap3>=1.0"],` ? As the first number is passed to pip and the second is used to check at runtime